.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
   - 2.0.0
   - 2.1.0
   - 2.1.1
```

Comment:
Also nto sure why check for 2.1.0 and 2.1.1 something breaking changed there?
Most of our apps are at 2.1.5 or 2.1.6